### PR TITLE
docs: minor improvements to tuning guide

### PR DESCRIPTION
**Pull request by borkmann**

### CODE REVIEW

File: Documentation/operations/performance/tuning.rst [hunk 1]
```diff
@@ -17,16 +17,18 @@ eBPF Host-Routing
...
-in Cilium 1.9 to fully bypass iptables. This option is automatically enabled
-if your kernel supports it. To validate whether your installation is running
-with eBPF host-routing, run ``cilium status`` in any of the Cilium pods and
-look for the line reporting the status for "Host routing".
+in Cilium 1.9 to fully bypass iptables and the upper host stack, and to achieve
+a faster network namespace switch compared to regular veth device operation.
+This option is automatically enabled if your kernel supports it. To validate
+whether your installation is running with eBPF host-routing, run ``cilium status``
+in any of the Cilium pods and look for the line reporting the status for
+"Host routing".
```
Comment (jibi) [inline Documentation/operations/performance/tuning.rst]: nit:
```suggestion
"Host Routing".
```
(and perhaps we can say that it should set to `BPF`?)

File: Documentation/operations/performance/tuning.rst [hunk 2]
```diff
@@ -38,12 +40,11 @@ look for the line reporting the status for "Host routing".
...
 
-Even when routing is performed by Cilium using eBPF host-routing, network
-packets still traverse the regular network stack in the network namespace
-of the container and iptables can again add significant cost. This traversal
-cost can be minimized by disabling the connection tracking requirement
-for packets forwarded by Cilium, thus bypassing the iptables connection
-tracker.
+For the case when eBPF Host-Routing cannot be used and thus network packets
+still need to traverse the regular network stack in the host namespace, then
+iptables can again add significant cost. This traversal cost can be minimized
```
Comment (jibi) [inline Documentation/operations/performance/tuning.rst]:
```suggestion
still need to traverse the regular network stack in the host namespace,
iptables can again add a significant cost. This traversal cost can be minimized
```

File: Documentation/operations/performance/tuning.rst [hunk 3]
```diff
@@ -38,12 +40,11 @@ look for the line reporting the status for "Host routing".
...
+by disabling the connection tracking requirement for packets forwarded by Cilium,
```
Comment (jibi) [inline Documentation/operations/performance/tuning.rst]: > for packets forwarded by Cilium

nit: to keep this in line with the documentation of the `--install-no-conntrack-iptables-rules` option ("Install Iptables rules to skip netfilter connection tracking on all pod traffic") maybe we can change it to "for all pod traffic"? (as also hostns -> pod traffic is matched by these rules, but this traffic is not entirely forwarded by Cilium)

File: Documentation/operations/performance/tuning.rst [hunk 4]
```diff
@@ -245,6 +247,15 @@ performance:
...
 
-We highly recommend to pin the NIC interrupts to specific CPUs. See `this script
-<https://github.com/borkmann/netperf_scripts/blob/master/set_irq_affinity>`_
-for details and pointers on how to achieve this.
+We highly recommend to pin the NIC interrupts to specific CPUs in order to
+allow for maximum workload isolation!
+
+See `this script <https://github.com/borkmann/netperf_scripts/blob/master/set_irq_affinity>`_
+for details and initial pointers on how to achieve this, pinning the queues
+can potentially vary in setup between different drivers.
```
Comment (jibi) [inline Documentation/operations/performance/tuning.rst]:
```suggestion
for details and initial pointers on how to achieve this. Note that pinning the queues
can potentially vary in setup between different drivers.
```

**Review by jibi — APPROVED** [review]
Looks good :rocket: just a few minor nits

**Review by qmonnet — APPROVED** [review]
Looks good to me, thanks!

**Review by jrfastab — APPROVED** [review]

File: Documentation/operations/performance/tuning.rst (hunk 3 above)
Comment (borkmann) [reply Documentation/operations/performance/tuning.rst]: I've changed into `This traversal cost can be minimized by disabling the connection tracking requirement for all Pod traffic, thus bypassing the iptables connection tracker.`

**Pull request merged**